classes/help.php

```diff
@@ -260,36 +259,36 @@ function print_page() {
 
 										if ( ! url_has_ftps_for_item( $simplemap_ps ) ) : ?>
 
-											<h4><?php printf( __( 'SimpleMap Premium Support Benefits', 'SimpleMap' ), esc_attr( site_url() ) ); ?></h4>
+											<h4><?php printf( __( 'SimpleMap Premium Support Benefits', 'simplemap' ), esc_attr( site_url() ) ); ?></h4>
 											<p>
-												<?php printf( __( 'SimpleMap now offers a premium support package for the low cost of %s per year per domain.', 'SimpleMap' ), '$30.00 USD' ); ?>
+												<?php printf( __( 'SimpleMap now offers a premium support package for the low cost of %s per year per domain.', 'simplemap' ), '$30.00 USD' ); ?>
```

Comment:
__[code_sniffer]__
* Spaces must be used to indent lines; tabs are not allowed
* Space after opening parenthesis of function call prohibited
* Expected 0 spaces before closing bracket; 1 found
* Line exceeds 120 characters; contains 193 characters

<!-- Comment by [SideCI](https://sideci.com) -->